### PR TITLE
Do not keep track of Cargo.toml files to avoid missing new ones

### PR DESCRIPTION
**Pull request by D4nte**
Because I already almost did the mistake with PR #2226.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -90,9 +92,7 @@ MODIFIED_TYPESCRIPT_FILES = $(filter %.ts %.json %.yml,$(MODIFIED_FILES))
 
 format: install_rustfmt install_tomlfmt
 	$(CARGO_NIGHTLY) fmt -- --files-with-diff | xargs -I{} git add {}
-	RUST_LOG=error $(CARGO) tomlfmt -p Cargo.toml && git add Cargo.toml
-	RUST_LOG=error $(CARGO) tomlfmt -p cnd/Cargo.toml && git add cnd/Cargo.toml
-	RUST_LOG=error $(CARGO) tomlfmt -p libp2p-comit/Cargo.toml && git add libp2p-comit/Cargo.toml
+	@$(foreach file,$(CARGO_TOML_FILES),$(CARGO) tomlfmt -p $(file) && git add $(file);)
```
Comment (thomaseizinger) [inline Makefile]: :mega: I think you dropped the `RUST_LOG` now right? Can you add that back please?

Comment (D4nte) [reply Makefile]: Can you explain the utility of the `RUST_LOG` here?

Comment (thomaseizinger) [reply Makefile]: To silence the output of tomlfmt.

Comment (D4nte) [reply Makefile]: Have you tested locally? I achieve the wanted result with the `@` at the start of the line.

Comment (thomaseizinger) [reply Makefile]: I did not!

I trust you that you figured out this Makefile voodoo :)

File: Makefile [hunk 2]
```diff
@@ -15,6 +15,8 @@ INSTALLED_COMPONENTS = $(shell $(RUSTUP) component list --installed --toolchain
 INSTALLED_NIGHTLY_COMPONENTS = $(shell $(RUSTUP) component list --installed --toolchain $(NIGHTLY_TOOLCHAIN))
 AVAILABLE_CARGO_COMMANDS = $(shell $(CARGO) --list)
 
+CARGO_TOML_FILES = $(shell ls **/Cargo.toml)
```
Comment (thomaseizinger) [inline Makefile]: Actually, I think you shouldn't even need this `ls`. The Makefile should expand the glob directly no?

**Pull request merged**